test/cmdlineTests/evmasm_transient_storage_state_variable/output

```diff
@@ -0,0 +1,95 @@
+
+======= evmasm_transient_storage_state_variable/input.sol:C =======
+EVM assembly:
+  mstore(0x40, 0x80)
+  0x2a
+  0x00
+  tstore
+  callvalue
+  dup1
+  iszero
+  tag_1
+  jumpi
+  0x00
+  dup1
+  revert
```

Comment:
Needs an expectation update after the push0 constant optimizer merge (`revert(0x00, 0x00)`).

Reply:
Fixed.